### PR TITLE
Updating Modules and Activities Response

### PR DESCRIPTION
**Pull request by avkaur**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by avkaur** [comment]
@vrdmr Please take a look.

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
           "description": "Gets or sets a Boolean value that indicates true if the parameter is dynamic."
         },
         "position": {
-          "type": "boolean",
+          "type": "integer",
```
Comment (marstr) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json]: This is a breaking change for code gen in most or all languages, and this swagger is in the `stable` folder.

**Review by marstr — CHANGES_REQUESTED** [review]
Before we get too far, could you address the breaking changes?

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json (hunk 1 above)
Comment (vrdmr) [reply specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json]: Our SDK is still in preview mode and open for breaking changes (https://www.nuget.org/packages/Microsoft.Azure.Management.Automation/3.0.1-preview). Also, from what I understood, the service returns integer and not boolean, which is why we corrected this and changed to integer type from boolean. 

@avkaur Could you please confirm.

**Review by marstr — APPROVED** [review]

**Comment by marstr** [comment]
Are these changes deployed to the server and ready for me to merge the swagger?

**Comment by marstr** [comment]
gentle ping, @avkaur

**Comment by marstr** [comment]
I'm going to close this PR as stale. Feel free to comment on this thread again and I'll open it right up, or open a new PR.

**Comment by vrdmr** [comment]
Hi @marstr,

The changes are now deployed. Could you please reopen the PR and merge the changes.

Thanks,
Varad

@avkaur

**Comment by marstr** [comment]
@vrdmr, 

It looks like there are some conflicts. Please fix them before I merge.

**Comment by marstr** [comment]
Closing as per @vrdmr's recommendation above.

**Pull request closed without merging**